drawBot/context/baseContext.py

```diff
@@ -265,6 +265,8 @@ def text(self, txt, offset=None, font=_FALLBACKFONT, fontSize=10, align=None):
 
         Optionally `txt` can be a `FormattedString`.
         """
+        if not isinstance(txt, (basestring, FormattedString)):
+            raise ValueError("depythonifying 'str' or 'FormattedString', got '%s'" % type(txt).__name__)
```

Comment:
Should be `TypeError`, and the message should say "expected 'str' or 'FormattedString', got '%s' ..."

Reply:
Made ValueError as what this is what pyobjc is raising... TypeError is idd better